<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QTreeWidget" name="treeWidget2">
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QTreeView {
    alternate-background-color: yellow;
}

QTreeView {
 /*   show-decoration-selected: 1;*/
}

QTreeView::item {
	
	background-color: rgb(0, 170, 255);
/*
     border: 1px solid #d9d9d9;
    border-top-color: transparent;
    border-bottom-color: transparent;
*/
}

QTreeView::item:hover {
	background-color: rgb(160, 162, 31);
/*
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #e7effd, stop: 1 #cbdaf1);
    border: 1px solid #bfcde4;
*/
}

QTreeView::item:selected {
    border: 1px solid #567dbc;
}

QTreeView::item:selected:active{
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #6ea1f1, stop: 1 #567dbc);
}

QTreeView::item:selected:!active {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #6b9be8, stop: 1 #577fbf);
}

QTreeView::branch {
        background: palette(base);
}

QTreeView::branch:has-siblings:!adjoins-item {
        background: rgb(0, 170, 255);
}

QTreeView::branch:has-siblings:adjoins-item {
        background: rgb(0, 170, 255);
}

QTreeView::branch:!has-children:!has-siblings:adjoins-item {
      background: rgb(0, 170, 255);
}

QTreeView::branch:closed:has-children:has-siblings {
	border-image: url(:/images/payloadIcon/satellite.png);
}

QTreeView::branch:has-children:!has-siblings:closed {
        background: gray;
}

QTreeView::branch:open:has-children:has-siblings {
	border-image: url(:/images/payloadIcon/earth.png);
}

QTreeView::branch:open:has-children:!has-siblings {
        background: green;
}
</string>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>true</bool>
         </property>
         <property name="iconSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="indentation">
          <number>20</number>
         </property>
         <property name="rootIsDecorated">
          <bool>true</bool>
         </property>
         <attribute name="headerVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="headerDefaultSectionSize">
          <number>120</number>
         </attribute>
         <attribute name="headerHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="headerShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>1</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>2</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>First1</string>
          </property>
          <item>
           <property name="text">
            <string>Second1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Second2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Second3</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>First2</string>
          </property>
          <item>
           <property name="text">
            <string>Second1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Second2</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>First3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>First4</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="CustomWidget" name="widget_2" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 0, 0);</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="CustomTreeWidget" name="treeWidget">
         <property name="styleSheet">
          <string notr="true">QTreeView {
    alternate-background-color: yellow;
}

QTreeView {
 /*   show-decoration-selected: 1;*/
}

QTreeView::item {
	
	background-color: rgb(0, 170, 255);
/*
     border: 1px solid #d9d9d9;
    border-top-color: transparent;
    border-bottom-color: transparent;
*/
}

QTreeView::item:hover {
	background-color: rgb(160, 162, 31);
/*
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #e7effd, stop: 1 #cbdaf1);
    border: 1px solid #bfcde4;
*/
}

QTreeView::item:selected {
    border: 1px solid #567dbc;
}

QTreeView::item:selected:active{
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #6ea1f1, stop: 1 #567dbc);
}

QTreeView::item:selected:!active {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #6b9be8, stop: 1 #577fbf);
}

QTreeView::branch {
        background: palette(base);
}

QTreeView::branch:has-siblings:!adjoins-item {
        background: rgb(0, 170, 255);
}

QTreeView::branch:has-siblings:adjoins-item {
        background: rgb(0, 170, 255);
}

QTreeView::branch:!has-children:!has-siblings:adjoins-item {
      background: rgb(0, 170, 255);
}

QTreeView::branch:closed:has-children:has-siblings {
	border-image: url(:/images/payloadIcon/satellite.png);
}

QTreeView::branch:has-children:!has-siblings:closed {
        background: gray;
}

QTreeView::branch:open:has-children:has-siblings {
	border-image: url(:/images/payloadIcon/earth.png);
}

QTreeView::branch:open:has-children:!has-siblings {
        background: green;
}
</string>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>1</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>2</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QTreeWidget" name="treeWidget3">
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>true</bool>
         </property>
         <property name="iconSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="indentation">
          <number>20</number>
         </property>
         <property name="rootIsDecorated">
          <bool>true</bool>
         </property>
         <attribute name="headerVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="headerDefaultSectionSize">
          <number>120</number>
         </attribute>
         <attribute name="headerHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="headerShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>1</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>First1</string>
          </property>
          <item>
           <property name="text">
            <string>Second1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Second2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Second3</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>First2</string>
          </property>
          <item>
           <property name="text">
            <string>Second1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Second2</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>First3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>First4</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>delete3</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>829</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>customtreewidget.h</header>
  </customwidget>
  <customwidget>
   <class>CustomWidget</class>
   <extends>QWidget</extends>
   <header>UiTableWidgetItemSecond.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
